### PR TITLE
Glow Stickers

### PR DESCRIPTION
**Pull request by Sovexe**
## About the PR 
* Adds Glow Stickers
* stickers that glow when stuck to things
* can be unstuck and restuck
* do not give off any light when not stuck to something
* Adds one box to Oshan mining hanger. No box for manta yet.
* Box holds 20 stickers

image

## Why's this needed? 
* Let miners place down glowing markers in the trench for lighting and navigational purposes

## Changelog

```changelog
(u)Sovexe
(*)New item, glow stickers. Stickers that glow when stuck to something. Now provided to undersea miners.
```

### CODE REVIEW

**Comment by MatheusIsNotAvailable** [comment]
Maybe put them in a cargo crate?

File: code/obj/item/stickers.dm [hunk 1]
```diff
@@ -580,3 +580,72 @@
...
+	New()
+		. = ..()
+		color = rgb(col_r*255, col_g*255, col_b*255)
+		light_c = src.AddComponent(/datum/component/holdertargeting/simple_light, col_r*255, col_g*255, col_b*255, brightness*255)
+		light_c.update(0)
+
+	attack_hand(mob/user as mob)
+		user.lastattacked = user
+		if (src.attached)
+			if (user.a_intent == INTENT_HELP)
```
Comment (flrsh) [inline code/obj/item/stickers.dm:604]: I feel like grab would make more sense?

**Review by flrsh — COMMENTED** [review]
Very cool. Up to you if you want to put these in a QM crate, though I actually think spawning these on miners (`#ifdef UNDERWATER_MAP`) might be better, since that way they're guaranteed to have them and thus more likely to use them.

**Comment by ZeWaka** [comment]
Consider: manta &  OTHER_MAP

**Comment by Sovexe** [comment]
> Very cool. Up to you if you want to put these in a QM crate, though I actually think spawning these on miners (`#ifdef UNDERWATER_MAP`) might be better, since that way they're guaranteed to have them and thus more likely to use them. 

Giving them to miners could be good I'll look into that

File: code/obj/item/stickers.dm (hunk 1 above)
Comment (Sovexe) [reply code/obj/item/stickers.dm:604]: these mimic sticky notes which use help

**Pull request merged**